planet/Decoration.ocd/Village.ocd/FenceRustic.ocd/Script.c

```diff
@@ -2,7 +2,11 @@
 	@author Dustin Neß (dness.de)
 */
 
-protected func Construction()
+public func Definition(proplist def)
 {
-	SetProperty("MeshTransformation", Trans_Mul(Trans_Rotate(RandomX(-35,35),0,10), Trans_Scale(60)));
-}
\ No newline at end of file
+	def.MeshTransformation = Trans_Scale(60);
+}
+
+public func Construction() {
+	SetProperty("MeshTransformation", Trans_Mul(Trans_Rotate(RandomX(-35,35),0,10), GetID().MeshTransformation));
+}
```

Comment:
By the way, the "protected func" engine callbacks can be reduced to simply "func". "public func" is necessary only if you want this function to be displayed in the editor.